python/runtime/local/submitter.py

```diff
@@ -160,6 +160,16 @@ def submit_local_explain(datasource,
                  model=model)
 
 
+def submit_local_run(
+    datasource,
+    select,
+    image_name,
+    params,
+    into):
+    print("Execute local run. datasource:{}, select: {}, image_name: {}, params: {}, into: {}".format(
```

Comment:
Add a space behind `datasource:` to unify the format.

Reply:
Good catch, updated.